Dockerfile

```diff
@@ -0,0 +1,36 @@
+ARG BASE_IMAGE
+
+FROM golang:1.13.5-alpine3.11 as builder
+
+RUN set -x && \
+    apk add --no-cache git gcc libc-dev && \
+    go get bitbucket.org/liamstask/goose/cmd/goose
+
```

Comment:
Maybe better to remove the line for consistency.

Reply:
Okay, thanks!